### ISSUE TITLE
Cannot run jaspr in Linux Mint

### ISSUE THREAD

**Issue by mjablecnik** [schultek/jaspr#5]
Hello I am trying run in Linux Mint command: `dart run jaspr serve -v`

But it return me:
```
martin at probook-pc my_web_app >>> dart run jaspr serve -v
Building package executable...
Built jaspr:jaspr.
Starting jaspr development server...
Failed to build webdev:webdev:
/home/martin/.pub-cache/hosted/pub.dartlang.org/dwds-12.1.1/lib/src/services/chrome_proxy_service.dart:540:24: Error: The method 'ChromeProxyService.getSourceReport' has fewer named arguments than those of overridden method 'VmServiceInterface.getSourceReport'.
  Future<SourceReport> getSourceReport(String isolateId, List<String> reports,
                       ^
/home/martin/.pub-cache/hosted/pub.dartlang.org/vm_service-8.3.0/lib/src/vm_service.dart:846:24: Context: This is the overridden method ('getSourceReport').
  Future<SourceReport> getSourceReport(
                       ^
/home/martin/.pub-cache/hosted/pub.dartlang.org/dwds-12.1.1/lib/src/services/chrome_proxy_service.dart:540:24: Error: The method 'ChromeProxyService.getSourceReport' doesn't have the named parameter 'libraryFilters' of overridden method 'VmServiceInterface.getSourceReport'.
  Future<SourceReport> getSourceReport(String isolateId, List<String> reports,
                       ^
/home/martin/.pub-cache/hosted/pub.dartlang.org/vm_service-8.3.0/lib/src/vm_service.dart:846:24: Context: This is the overridden method ('getSourceReport').
  Future<SourceReport> getSourceReport(
                       ^
Failed to build webdev:webdev:
/home/martin/.pub-cache/hosted/pub.dartlang.org/dwds-12.1.1/lib/src/services/chrome_proxy_service.dart:540:24: Error: The method 'ChromeProxyService.getSourceReport' has fewer named arguments than those of overridden method 'VmServiceInterface.getSourceReport'.
  Future<SourceReport> getSourceReport(String isolateId, List<String> reports,
                       ^
/home/martin/.pub-cache/hosted/pub.dartlang.org/vm_service-8.3.0/lib/src/vm_service.dart:846:24: Context: This is the overridden method ('getSourceReport').
  Future<SourceReport> getSourceReport(
                       ^
/home/martin/.pub-cache/hosted/pub.dartlang.org/dwds-12.1.1/lib/src/services/chrome_proxy_service.dart:540:24: Error: The method 'ChromeProxyService.getSourceReport' doesn't have the named parameter 'libraryFilters' of overridden method 'VmServiceInterface.getSourceReport'.
  Future<SourceReport> getSourceReport(String isolateId, List<String> reports,
                       ^
/home/martin/.pub-cache/hosted/pub.dartlang.org/vm_service-8.3.0/lib/src/vm_service.dart:846:24: Context: This is the overridden method ('getSourceReport').
  Future<SourceReport> getSourceReport(
                       ^
```

How can I fix it?

My Dart version is:
```
Dart SDK version: 2.16.1 (stable) (Tue Feb 8 12:02:33 2022 +0100) on "linux_x64"
```
And linux version:
```
Distributor ID: LinuxMint
Description:    Linux Mint 19.3 Tricia
Release:        19.3
Codename:       tricia
```

And here are my dependencies:
```
martin at probook-pc my_web_app >>> dart pub upgrade
Resolving dependencies... (2.4s)
  _fe_analyzer_shared 34.0.0 (40.0.0 available)
  analyzer 3.2.0 (4.1.0 available)
  archive 3.3.0
  args 2.3.1
  async 2.9.0
  bazel_worker 1.0.1
  binary_codec 2.0.3
  browser_launcher 1.1.0
  build 2.3.0
  build_config 1.0.0
  build_daemon 3.1.0
  build_modules 4.0.4 (4.0.5 available)
  build_resolvers 2.0.6 (2.0.9 available)
  build_runner 2.1.11
  build_runner_core 7.2.3
  build_web_compilers 3.2.3
  built_collection 5.1.1
  built_value 8.3.2
  checked_yaml 2.0.1
  cli_util 0.3.5
  code_builder 4.1.0
  collection 1.16.0
  convert 3.0.2
  crypto 3.0.2
  csslib 0.17.2
  dart_style 2.2.1 (2.2.3 available)
  dds 2.2.1
  dds_service_extensions 1.3.0
  devtools_shared 2.13.1 (2.14.0 available)
  domino 0.8.3
  dwds 12.1.1 (14.0.3 available)
  file 6.1.2
  fixnum 1.0.1
  frontend_server_client 2.1.3
  glob 2.0.2
  graphs 2.1.0
  hotreloader 3.0.4
  html 0.15.0
  http 0.13.4
  http_multi_server 3.2.0
  http_parser 4.0.1
  io 1.0.3
  jaspr 0.1.3
  js 0.6.4
  json_annotation 4.5.0
  json_rpc_2 3.0.1
  lints 1.0.1 (2.0.0 available)
  logging 1.0.2
  matcher 0.12.11
  meta 1.8.0
  mime 1.0.2
  package_config 2.0.2
  path 1.8.2
  pedantic 1.11.1
  pool 1.5.0
  protobuf 2.0.1
  pub_semver 2.1.1
  pubspec_parse 1.2.0
  scratch_space 1.0.1
  shelf 1.3.0
  shelf_packages_handler 3.0.0
  shelf_proxy 1.0.1
  shelf_static 1.1.0
  shelf_web_socket 1.0.1
  source_maps 0.10.10
  source_span 1.9.0
  sse 4.1.0
  stack_trace 1.10.0
  stream_channel 2.1.0
  stream_transform 2.0.0
  string_scanner 1.1.1
  term_glyph 1.2.0
  timing 1.0.0
  typed_data 1.3.1
  usage 4.0.2
  uuid 3.0.6
  vm_service 8.3.0
  watcher 1.0.1
  web_socket_channel 2.2.0
  webdev 2.7.8 (2.7.9 available)
  webkit_inspection_protocol 1.1.0
  yaml 3.1.1
No dependencies changed.
9 packages have newer versions incompatible with dependency constraints.
Try `dart pub outdated` for more information.
```

**Comment by mjablecnik** [comment]
After upgrade dart version to 2.17.1 and upgrade dependencies this error dissapeared.

**Comment by mjablecnik** [comment]
I think you should support minimal Dart version 2.17.0 instead of 2.16.0 in pubspec.yaml of your generated new project:
![image](https://user-images.githubusercontent.com/2686906/172185572-d78bf8fa-964f-4579-9776-3b80cd653856.png)

**Comment by schultek** [comment]
Fixed in `v0.1.4`
